opentelemetry-api/src/opentelemetry/attributes/__init__.py

```diff
@@ -108,3 +111,73 @@ def _create_immutable_attributes(
     attributes: types.Attributes,
```

Comment:
We probably don't need this function anymore right?

Reply:
removed